winpr/include/winpr/custom-crypto.h

```diff
@@ -185,38 +185,16 @@ extern "C"
 #define WINPR_CIPHER_AES_128_GCM 14
 #define WINPR_CIPHER_AES_192_GCM 15
 #define WINPR_CIPHER_AES_256_GCM 16
-#define WINPR_CIPHER_CAMELLIA_128_ECB 17
```

Comment:
why remove these?

Reply:
mbedtls deprecated all DES and blowfish ciphers, as for Camellia, they're not deprecated yet but they were never used, so I decided to just reduce the set of cipher IDs we map to a more reasonable subset

Reply:
we´ve released 3.0 with these, so I´m for playing it safe here and just change what is really required. (yes, should not be used anywhere, but you never know)